web/modules/custom/form_api/form_api.module

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * Implements hook_install().
+ */
+function form_api_install() {
+  $default_config = \Drupal::configFactory()->getEditable('form_api.settings');
+  $default_config->set('full_name', 'Souvik Banerjee');
+  $default_config->set('phone_number', '+919873214560');
+  $default_config->set('email', 'this@gmail.com');
+  $default_config->set('gender', 'male');
+  $default_config->save();
```

Comment:
Why add this?

Reply:
This was setting the config file on install using hook, now have removed it.